legacy/packages.ini

```diff
@@ -38,3 +38,19 @@ php7.3-gd = buster
 php7.3-mbstring = buster
 php7.3-pgsql = buster
 php7.3-xml = buster
+
+# Focal
+[php7.4]
+libapache2-mod-php7.4 = focal
+php7.4 = focal
+php7.4-apcu = focal
+php7.4-apcu-bc = focal
+php7.4-bcmath = focal
+php7.4-cli = focal
+php7.4-common = focal
+php7.4-curl = focal
+php7.4-dev = focal
+php7.4-gd = focal
+php7.4-mbstring = focal
+php7.4-pgsql = focal
+php7.4-xml = buster, focal
```

Comment:
I think we missed the extra `buster,` here right ?

Reply:
Yup, seems so